lib/meilisearch/rails/multi_search.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'multi_search/result'
+
+module MeiliSearch
+  module Rails
+    class << self
+      def multi_search(searches)
```

Comment:
Placed the multi search method as `MeiliSearch::Rails.multi_search` for the time being, but it does not feel like we should expect our users to write it all out.

Maybe `ApplicationRecord.multi_search`, or give all models the same, generic, `multi_search` method?

Reply:
I think for convenience, we could add it to `ApplicationRecord` indeed.